docs/configuration.md

```diff
@@ -172,6 +175,55 @@ from the table above:
 export GRAFANA_DOTNET_DISABLE_INSTRUMENTATIONS="Process,NetRuntime"
 ```
 
+## Resource detector configuration
+
+### Specifying resource detectors
+
+Default resource detectors can be overridden by removing them from the `ResourceDetectors`
+set in the configuration and specifying which resource detectors to enable:
+
+```csharp
+using var tracerProvider = Sdk.CreateMeterProviderBuilder()
+    .UseGrafana(config =>
+    {
+        config.ResourceDetectors.Clear(ResourceDetector.Host);
+        config.ResourceDetectors.Add(ResourceDetector.Process);
+    })
+    .Build();
+```
+
+Alternatively, resource detectors can be specified by the environment
+variable `GRAFANA_DOTNET_RESOURCEDETECTORS`. This variable can be
+populated with a comma-separated list of resource detector identifiers:
+
+```sh
+export GRAFANA_DOTNET_RESOURCEDETECTORS="Process"
+```
+
+### Disabling resource detectors
+
+By default, `Host`, `Process`, and `ProcessRuntime` resource detectors are enabled.
+Resource detectors can be disabled by removing them from the `ResourceDetectors`
+set in the configuration:
+
+```csharp
+using var tracerProvider = Sdk.CreateMeterProviderBuilder()
+    .UseGrafana(config =>
+    {
+        config.ResourceDetectors.Remove(ResourceDetector.Host);
```

Comment:
is clear and remove the same?

Reply:
`Clear` empties all values from collection. `Remove` is for a single, specific value.